server/core/config/raw/policies.go

```diff
@@ -52,16 +52,16 @@ func (o PolicyOwners) ToValid() valid.PolicyOwners {
 }
 
 type PolicySet struct {
-	Path   string       `yaml:"path" json:"path"`
-	Source string       `yaml:"source" json:"source"`
-	Name   string       `yaml:"name" json:"name"`
-	Owners PolicyOwners `yaml:"owners,omitempty" json:"owners,omitempty"`
+	Path   string `yaml:"path" json:"path"`
+	Source string `yaml:"source" json:"source"`
+	Name   string `yaml:"name" json:"name"`
+	Owner  string `yaml:"owner,omitempty" json:"owner,omitempty"`
```

Comment:
Was this "owners" field used previously?

Reply:
No, it wasn't used yet.